### ISSUE TITLE
Outline view does not work

### ISSUE THREAD

**Issue by edertone** [angelozerr/typescript.java#174]
First of all, this is the best typescript plugin for eclipse right now. Great work!

There is a small problem. If you use outline view (ctrl + o) it appears empty, instead of showing the current class methods and properties.

thanks!

**Comment by angelozerr** [comment]
> First of all, this is the best typescript plugin for eclipse right now. Great work!

Thanks!

The Quick outline view is unable when tsserver is available. If you close your ts editor and you reopen it, is it working?

**Comment by edertone** [comment]
Yes, you're right. Outline view works after closing all the opened ts files and opening them again

**Comment by angelozerr** [comment]
I must find time how to fix this bug.

**Comment by edertone** [comment]
Not a big problem knowing the workaround!

**Comment by wolfgang-ch** [comment]
There is a "shortcut" for reopening a file:

- Close with `<Ctrl> + w`
- Reopen with `<Alt> + left cursor` (this navigates the history)
